modules/openapi-generator/src/main/resources/typescript-fetch/modelOneOf.mustache

```diff
@@ -11,4 +13,13 @@ import {
  * {{{description}}}{{/description}}
  * @export
  */
-export type {{classname}} = {{#oneOf}}{{{.}}}{{^-last}} | {{/-last}}{{/oneOf}};
\ No newline at end of file
+export type {{classname}} = {{#oneOf}}{{{.}}}{{^-last}} | {{/-last}}{{/oneOf}};
+
+export function {{classname}}FromJSON(json: any): {{classname}} {
+    //TODO to implement
```

Comment:
I would remove this as long as this has not yet been implemented

Reply:
i reverted it. in fact typescript-fetch doesn't support completly this feature cause a JSON parser function is necessary to the oneOf Model for the api.
But i don't know how to implement it.